web/js/utils-0.0.23.js

```diff
@@ -2038,3 +2266,20 @@ function textInputHasFocus() {
         document.activeElement.nodeName === 'INPUT' &&
         document.activeElement.type === 'text';
 }
+
+var htmlEscapeMap = {
```

Comment:
can this be a local variable in the escapeHtml?

Reply:
Yes. I will change it.